Polyhedron/demo/Polyhedron/Plugins/PMP/RemoveNeedlesDialog.ui

```diff
@@ -89,18 +89,27 @@
         <string>Do not collapse edges begger than this threshold.</string>
        </property>
        <property name="title">
-        <string>Collapse Length Threshold</string>
+        <string>Needle Threshold: Maximum Length of Collapsable Edges (0 means Infinity)</string>
```

Comment:
```suggestion
        <string>Do not collapse needles (according to the criterion above) whose shortest edge has a length greater than:</string>
```

Reply:
and I would put a checkbox to enable/disable (maybe for all three fields, actually?).

Reply:
I won't do it, it is too much work as this is not possible in the function.